### PR TITLE
chore: remove buy button from mobile header

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

buy/sell has lived in the mobile footer for a long while now and this is redundant, assymmetric, and ugly.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

n/a

## Risk

basically zero - removing button from the mobile header

### Engineering

* see screenshots

### Operations

* see screenshots

## Screenshots (if applicable)

before

image

after

image

### CODE REVIEW

**Review by gomesalexandre — APPROVED** [review]
I like red lines
